blueflood-core/src/test/java/com/rackspacecloud/blueflood/service/ConfigurationTest.java

```diff
@@ -270,6 +278,337 @@ public void testGettingValuesCreatesOriginals() {
         }
     }
 
+    @Test
+    public void testOriginals1() {
```

Comment:
Can we use a more meaningful test names other than testOriginal1 and testOriginal2?